### PR TITLE
Implementingthecollectionfeature

### PR DESCRIPTION
**Pull request by folusho-adeyemi**
## Description

Implementing the collection feature..
I am yet to add more feature, like to create new collections, delete collections and also delete items from collections.

## Milestones
Milestone 2

## Test Plan
<div>
    <a href="https://www.loom.com/share/d457b506af914e17839e436b48270011">
      <p>FashionConnect - 19 July 2023 - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/d457b506af914e17839e436b48270011">
      
    </a>
  </div>

### CODE REVIEW

File: FashionConnect_ui/src/App.jsx [hunk 1]
```diff
@@ -30,18 +30,60 @@ function App() {
...
+    fetchProducts();
+  }, [collections]);
+
+  const fetchProductDetails = async (productId) => {
+    try {
+      const diff_products = await fetch(`http://localhost:3000/products/${productId}`);
+      const product = await diff_products.json();
+      return product;
+    } catch (error) {
+      console.error('Error fetching product details:', error);
```
Comment (safirshahbaaz) [inline FashionConnect_ui/src/App.jsx:71]: Let's clean up all the console error logs please.

Comment (folusho-adeyemi) [reply FashionConnect_ui/src/App.jsx:71]: oh okay.
I will remove them as from now

**Review by safirshahbaaz — APPROVED** [review]
Overall looks good! Let's try to break up commits and PRs into smaller sizes. For example: we can structure this large PR as:
1. PR-1: Changes related to adding the new DB schema and endpoints
2. PR-2: Changes related to frontend code that interacts with the new endpoints

**Comment by folusho-adeyemi** [comment]
> Overall looks good! Let's try to break up commits and PRs into smaller sizes. For example: we can structure this large PR as:
> 
> 1. PR-1: Changes related to adding the new DB schema and endpoints
> 2. PR-2: Changes related to frontend code that interacts with the new endpoints

Okay will take note of that from now on.

**Pull request merged**